tests/unit/s2n_random_test.c

```diff
@@ -765,6 +765,17 @@ static int s2n_random_test_case_failure_cb(struct random_test_case *test_case)
     return EXIT_SUCCESS;
 }
 
+static int s2n_random_noop_destructor_test_cb(struct random_test_case *test_case)
```

Comment:
Should this file be `s2n_fork_test` or something so it's less weird if we want to add other kinds of forking tests here, like this new one?

Reply:
No, this test is still specifically testing s2n_random.c. The problematic destructor was in s2n_random.c. That's why I put this new test case in s2n_random_test.c :)